config/ftbquests/quests/chapters/anvils_to_beat_you_black_red_and_blue.snbt

```diff
@@ -149,7 +149,7 @@
 					}
 					id: "tfc:metal/anvil/blue_steel"
 				}
-				title: "{gravitas.quest.caveman.coloured_anvil}"
+				title: "Coloured_Anvil"
```

Comment:
these entries inside the {} are pointers to the lang file at [kubeks\assets\lang\gravitas](https://github.com/AllTheMods/Gravitas2/blob/main/kubejs/assets/gravitas/lang/en_us.json) any changes relating to things like this should be edited there,  and likewise other changes to existing quests that do not already have a corresponding lang file entry, should get one, for pack translation compat

Reply:
Thanks for the info. Still learning the ropes so I very much appreciate the detail